src/docs/introduction/upgrading-to-stencil-three.md

```diff
@@ -62,6 +62,33 @@ export const config: Config = {
 };
 ```
 
+### Legacy Browser Support Fields Deprecated
+
+Several configuration options related to support for Safari <11, IE11, and Edge
+<19 have been marked as deprecated, and will be removed entirely in a future
+version of Stencil.
+
+#### `dynamicImportShim`
+
+The `extras.dynamicImportShim` option causes Stencil to include a polyfill for
+the [dynamic `import()`
+function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import)
+for use at runtime. The field is renamed to `__deprecated__dynamicImportShim`
+to indicate deprecation. To retain the previous behavior the new option can be
+set in your project's `stencil.config.ts`:
+
+```ts
+// stencil.config.ts
+import { Config } from '@stencil/core';
+export const config: Config = {
+  extras: {
+    __deprecated__dynamicImportShim: true
+  }
+};
+```
+
+This option will be removed entirely in a future version of Stencil.
```

Comment:
Nothing major, but is this redundant w/ the sentence on lines 68/69?

Reply:
maybe a bit...idk should we remove it? I'm down, why not keep things a little more concise.